R/deprecate.R

```diff
@@ -1,12 +1,52 @@
 #' These functions will be deprecated. Please use other functions instead.
 #' 
+#' @param x A
+#'   \code{\link[SummarizedExperiment:SummarizedExperiment-class]{SummarizedExperiment}}
+#'   object.
+#'   
+#' @param clust.col A single character value indicating the name of the 
+#'   \code{rowData} (or \code{colData}) where the data will be stored.
+#'   
+#' @param exprs_values a single \code{character} value for specifying which
+#'   assay to use for calculation.
+#'   (Please use \code{assay.type} instead.)
+#'   
+#' @param transposed Logical scalar, is x transposed with cells in rows?
+#' 
+#' @param type the type of measure used for the goodness of fit. One of
```

Comment:
This manual page should collect all deprecated functions. Some of them might have same parameter name but different meaning. So I'm wondering what happens then. --> I'm wondering if it is better to not list parameters for any of these functions since user's should not use them anyways (even though BIocCheck would tell otherwise)